src/docs/Architecture.mdx

```diff
@@ -61,4 +61,4 @@ For a high level overview of Theia's Architecture, see this document:
 
 [Multi-Language IDE Implemented in JS - Scope and Architecture](https://docs.google.com/document/d/1aodR1LJEF_zu7xBis2MjpHRyv7JKJzW7EWI9XRYCt48)
 
-<DocArrowNavigators prev='index' next='extensions'/>
\ No newline at end of file
+<DocArrowNavigators prev='index' prevTitle="Intro" next='extensions' nextTitle='Extensions'/>
```

Comment:
Ouch, I had no idea we were maintaining the links between pages manually.

I wonder how long it will take for one of these links to break (e.g. moved/renamed page, unreachable new page, mismatched page order, etc).

A much more robust solution would be to generate these prev/next links programmatically, from a single source of truth.

Reply:
I agree! I never thought how easily it can break sorry. I will soon make a PR to make this automated.